apps/barista/src/components/color-grid/color-grid.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Input, AfterContentInit } from '@angular/core';
+import { DtColors } from '@dynatrace/barista-components/theming';
+
+@Component({
+  selector: 'ba-color-grid',
+  templateUrl: 'color-grid.html',
+  styleUrls: ['color-grid.scss'],
+})
+export class BaColorGrid implements AfterContentInit {
```

Comment:
I just realized that apart from this color grid component we also need a wrapping component for more color-components that are not of the same shade (see e.g. the logo-page or all the color-subpages in Barista). This one is just a wrapper with no behavior but provides a grid styling (same as for this color-grid component) for the child elements.

If you have some time, you could add it to this PR (and maybe reuse styles) or create a separate issue for that. Thanks!